PowerShell/Deploy/SdkSync/jcapiToSupportSync.ps1

```diff
@@ -0,0 +1,107 @@
+            $ParameterContent = ($Params.Matches.Value | Where-Object { $_ -notlike '*DontShow*' -and $_ -notlike '*Limit*' -and $_ -notlike '*Skip*' })
+            $OutputType = ($FunctionContent | Select-String -Pattern:('(\[OutputType)(.*?)(\]\r)')).Matches.Value
+            $CmdletBinding = ($FunctionContent | Select-String -Pattern:('(\[CmdletBinding)(.*?)(\]\r)')).Matches.Value
+            If (-not [System.String]::IsNullOrEmpty($PSScriptInfo))
+            {
+                $PSScriptInfo = $PSScriptInfo.Replace($SdkPrefix, $JumpCloudModulePrefix)
+            }
+            # Build CmdletBinding
+            If (-not [System.String]::IsNullOrEmpty($OutputType)) { $CmdletBinding = "$($OutputType)`n$($IndentChar)$($CmdletBinding)" }
+            # Build $BeginContent, $ProcessContent, and $EndContent
+            $BeginContent = @()
+            $ProcessContent = @()
+            $EndContent = @()
+            # Build "Begin" block
+            $BeginContent += "$($IndentChar)$($IndentChar)Connect-JCOnline -force | Out-Null"
+            $BeginContent += "$($IndentChar)$($IndentChar)`$Results = @()"
+            # Build "Process" block
+            $ProcessContent += "$($IndentChar)$($IndentChar)`$Results = $($ModuleName)\$($CommandName) @PSBoundParameters"
+            # Build "End" block
+            $EndContent += "$($IndentChar)$($IndentChar)Return `$Results"
+            If (-not [System.String]::IsNullOrEmpty($BeginContent) -and -not [System.String]::IsNullOrEmpty($ProcessContent) -and -not [System.String]::IsNullOrEmpty($EndContent))
+            {
+                # Build "Function"
+                $NewScript = $FunctionTemplate -f $PSScriptInfo, $NewCommandName, $CmdletBinding, ($ParameterContent -join ",`n`n"), ($BeginContent -join "`n"), ($ProcessContent -join "`n"), ($EndContent -join "`n")
+                # Fix line endings
+                $NewScript = $NewScript.Replace("`r`n", "`n").Trim()
+                # Export the function
+                $OutputFilePath = "$OutputPath/$NewCommandName.ps1"
+                New-FolderRecursive -Path:($OutputFilePath) -Force
+                $NewScript | Out-File -FilePath:($OutputFilePath) -Force
+                # Validate script syntax
+                $ScriptAnalyzerResult = Invoke-ScriptAnalyzer -Path:($OutputFilePath) -Recurse -ExcludeRule PSShouldProcess, PSAvoidTrailingWhitespace, PSAvoidUsingWMICmdlet, PSAvoidUsingPlainTextForPassword, PSAvoidUsingUsernameAndPasswordParams, PSAvoidUsingInvokeExpression, PSUseDeclaredVarsMoreThanAssignments, PSUseSingularNouns, PSAvoidGlobalVars, PSUseShouldProcessForStateChangingFunctions, PSAvoidUsingWriteHost, PSAvoidUsingPositionalParameters
+                If ($ScriptAnalyzerResult)
+                {
+                    $ScriptAnalyzerResults += $ScriptAnalyzerResult
+                }
+            }
+            # Copy tests?
```

Comment:
Should we add a todo to copy tests from sdk to powershell module to test during build?